CONTRIBUTING.md

```diff
@@ -2,23 +2,21 @@
 
 Make sure you read: [how to report bugs](https://github.com/citybound/citybound/wiki/How-to-report-bugs)
 
-# Plan to contribute code?
-
-## Make sure to <a href="https://www.clahub.com/agreements/citybound/citybound">sign the Contributor License Agreement</a>.
-
-## [Have a look at the documentation](http://citybound.github.io/citybound)
-
-## Compiling Citybound from source 
+# Compiling Citybound from source 
 
-Currently Citybound is built with Rust `nightly-2017-01-08`
+Currently Citybound is built with Rust `nightly-2017-04-28`
 
 Recommended setup:
-* install [rustup](https://rustup.rs/)
+* Install [rustup](https://rustup.rs/) and [git](https://git-scm.com/)
 * `git clone https://github.com/citybound/citybound.git`
 * `cd citybound`
-* Windows: `rustup override add nightly-2017-01-08-x86_64-pc-windows-gnu`
-* MacOS: `rustup override add nightly-2017-01-08-x86_64-apple-darwin`
-* Linux: `rustup override add nightly-2017-01-08-x86_64-unknown-linux-gnu`
+* Windows:
+  * `rustup override add nightly-2017-04-28-x86_64-pc-windows-msvc` **(new!)**
+  * Install the [Visual C++ 2015 Build Tools](http://landinghub.visualstudio.com/visual-cpp-build-tools)
```

Comment:
This would only be needed if the developer doesn't have Visual Studio 2015 installed.